scripts/ci-footprint-and-performance/setup/001-configure-wp-lowlat.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -xeuo pipefail
+export PS4='+ $(date "+%T.%N") ${BASH_SOURCE}:$LINENO \011'
+
+export MANAGEMENT_CPUSET="0-5"
+export WORKLOAD_CPUSET="10-30"
+
+sudo mkdir -p /etc/microshift
+sudo tee /etc/microshift/config.yaml << EOF
```

Comment:
We would probably benefit from comments in each of the following config sections?